website/content/docs/beta-features.md

```diff
@@ -614,3 +612,5 @@ collections:
     # moving an existing entry will move the entire sub tree of the entry to the new location
     meta: { path: { widget: string, label: 'Path', index_file: 'index' } }
 ```
+
+**Note:** Nested collections work poperly only with `slug/index.md` directory structure. If you use `slug.md` approach, you might see some unwanted results.
```

Comment:
I'm not sure what `slug.md` approach means.
Could we rephrase it as `Nested collections expect the following directory structure....` and provide something like this:
```
content
└── pages
    ├── authors
    │   ├── author-1
    │   │   └── index.md
    │   └── index.md
    ├── index.md
    └── posts
        ├── hello-world
        │   └── index.md
        └── index.md
```
Will that make it more clear?

Reply:
@erezrokah thanks for your review. Your suggestion makes more sense, so I changed the content accordingly. I spent quite some time dealing with this before I realized that this structure is required. So I opened this PR to maybe save someone else's time.

Would it be possible to make this work with this sort of structure? It's what I meant with `slug.md`. If it's possible, I would be glad to open an issue, maybe even a pull request.

```
content
└── pages
    ├── authors
    │   ├── author-1.md
    │   └── index.md
```

Reply:
> Would it be possible to make this work with this sort of structure? It's what I meant with `slug.md`. If it's possible, I would be glad to open an issue, maybe even a pull request.

Opening an issue will be a great start. A pull request will be 🔥